cmd/refinery/main.go

```diff
@@ -164,23 +165,19 @@ func main() {
 		ForceAttemptHTTP2:   true,
 	}
 
-	genericMetricsRecorder := metrics.NewMetricsPrefixer("")
-	upstreamMetricsRecorder := metrics.NewMetricsPrefixer("libhoney_upstream")
-	peerMetricsRecorder := metrics.NewMetricsPrefixer("libhoney_peer")
```

Comment:
If I'm understanding all of these changes correctly, the `libhoney_` portion of the existing prefix goes away in this PR I _think_ regardless of whether the `transmit` (libhoney) or `direct_transmit` (all msgpack all the time) is used.

Will that name change for all metrics break current Refinery boards?

Reply:
Ugh, you are absolutely correct. I think it's appropriate to remove `libhoney_` prefix from the metrics produced by `direct_transmit` since it no longer uses `libhoney`. However, I also hear the concern about existing Refinery boards.

Maybe this is the reason we should do a major release so that we can modify existing metrics. What do you think?